misk-web/web/packages/@misk/core/src/ducks/simpleNetworkDucks.ts

```diff
@@ -39,43 +40,64 @@ export interface ISimpleNetworkPayload {
 }
 
 export interface IDispatchSimpleNetwork {
-  delete: (
+  simpleNetworkDelete: (
     tag: string,
     url: string,
     requestConfig?: AxiosRequestConfig
   ) => IAction<SIMPLENETWORK.DELETE, ISimpleNetworkPayload>
-  failure: (error: any) => IAction<SIMPLENETWORK.FAILURE, ISimpleNetworkPayload>
-  get: (
+  simpleNetworkFailure: (
```

Comment:
interesting, why are success and failure a part of these definitions?

Reply:
By convention, they seem to be included in the dispatcher object since they are a Redux action type. That being said, they are functions that are to my knowledge only every internally called so I did look into separating them for this release. I chose against it initially because of the significant refactoring and merge typing that would be required but it may make sense so that the exported dispatcher that developers use only holds APIs that they should use.

Reply:
What is a redux action type :-). And how do those relate to the standard http actions?